todo_handler/views.py

```diff
@@ -88,7 +87,7 @@ def register(request):
         if form.is_valid():
             user = form.save()
             login(request, user)  # Log the user in after registration
-            return redirect(reverse_lazy('login'))  # Redirect to the login page using reverse_lazy
```

Comment:
just instead of redirect after login also handle it from login and signup pages views